source/educators/references/course_development/create_problem.rst

```diff
@@ -66,7 +66,7 @@ types** or select Advanced problem types for a list of **advanced problem types*
   Options for scoring, hints, feedback and more can be entered.
 
 * For advanced problem types, the :ref:`advanced editor<Advanced Editor>` opens.
-  In this editor you use `EdX Open Learning XML Guide`_.
+  In this editor you use :ref:`Set Up Group Configuration for OLX Courses`.
```

Comment:
We haven't moved OLX yet. So it looks like what was done here is you've linked to one of the few files that do mention OLX. But you are pointing specifically to group configuration, not generic OLX intro. 

We should just link to the legacy link in this case, since it is easy to track that error .

You've reminded me that I have this old issue assigned to me. I'll add a note to this issue to take care of this link once OLX is moved: https://github.com/openedx/docs.openedx.org/issues/495

So, in summary, revert this back to the `EdX Open Learning XML Guide`_ link

Reply:
Done. Thanks for the review and the clarification.